src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/QRScanner/ExposureSubmissionQRScannerViewModel.swift

```diff
@@ -196,3 +190,8 @@ class ExposureSubmissionQRScannerViewModel: NSObject, AVCaptureMetadataOutputObj
 	private let captureDevice: AVCaptureDevice?
 
 }
+enum TorchMode {
+	case notAvailable
+	case lightOn
+	case ligthOff
+}
```

Comment:
moving it here doesn't seem to fit if it gets used in other classes too. Same for `QRScannerError`- maybe move them in a better place to indicate it gets reused

Reply:
Done, i also fixed the other comments